Makefile

```diff
@@ -1,42 +1,112 @@
-help: ## Display this help menu
-	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+# Executables (local)
+DOCKER_COMP = docker compose
 
+# Docker containers
+PHP_CONT = $(DOCKER_COMP) exec prestashop-git
+
+# Executables
+PHP      = $(PHP_CONT) php
+COMPOSER = $(PHP_CONT) composer
+SYMFONY  = $(PHP) bin/console
+
+# Misc
+.DEFAULT_GOAL = install
+.PHONY        : help docker-build docker-up docker-start docker-down docker-logs docker-sh docker-bash composer sf cc test test-unit test-integration assets assets-dev wait-assets admin front admin-default admin-new-theme front-core front-classic front-hummingbird install install-prestashop cs-fixer cs-fixer-dry phpstan scss-fixer es-linter
+
+## —— 🎵 🐳 PrestaShop Docker Makefile 🐳 🎵 ———————————————————————————————————
+help: ## Outputs this help screen
+	@grep -E '(^[a-zA-Z0-9\./_-]+:.*?##.*$$)|(^##)' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}{printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}' | sed -e 's/\[32m##/[33m/'
+
+## —— Docker 🐳 ————————————————————————————————————————————————————————————————
+docker-build: ## Builds the Docker images
+	COMPOSE_BAKE=true $(DOCKER_COMP) build --pull --no-cache
+
+docker-up: ## Start the docker hub in detached mode (no logs)
+	$(DOCKER_COMP) up --detach --force-recreate --remove-orphans
+
+docker-start: docker-build docker-up ## Build and start the containers
+
+docker-down: ## Stop the docker hub
+	@$(DOCKER_COMP) down --remove-orphans
+
+docker-logs: ## Show live logs
+	@$(DOCKER_COMP) logs --follow
+
+docker-sh: ## Connect to the PHP container
+	@$(PHP_CONT) sh
+
+docker-bash: ## Connect to the PHP container via bash so up and down arrows go to previous commands
+	@$(PHP_CONT) bash
```

Comment:
Maybe we should run the shell as `www-data` instead of `root`, the image already forwards the host’s UID/GID to the `www-data` user inside container using `USER_ID` / `GROUP_ID` build args [here](https://github.com/PrestaShop/PrestaShop/blob/9.0.x/.docker/Dockerfile#L18)), so dropping into a root shell can leave root‑owned files on the host. 
Using a runuser wrapper should keeps file permissions aligned with the host user : 
`runuser -g www-data -u www-data -- bash`

Reply:
Thanks for your report ! It is now up-tp-date with the changes you requested.